packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/StackDemo/StackDemo.tsx

```diff
@@ -1,14 +1,18 @@
 import React, { Component } from 'react';
-import { Stack, StackItem } from '@patternfly/react-core';
+import { Badge, Stack, StackItem } from '@patternfly/react-core';
 
 export class StackDemo extends Component {
   render() {
     return (
-      <Stack gutter="sm">
-        <StackItem>content</StackItem>
-        <StackItem isFilled>pf-m-fill</StackItem>
-        <StackItem>content</StackItem>
-      </Stack>
+      <React.Fragment>
+        <Badge>3</Badge>
```

Comment:
What I meant in my last reviews was use the `component` prop of Stack.  It is nice to exercise all props in the demos to verify there are no errors when the consumer tries to use them.